src/hubmap_translator.py

```diff
@@ -792,7 +793,7 @@ def add_calculated_fields(self, entity):
     # Upload: Just make it "Data Upload" for all uploads
     # Donor: "Donor"
     # Sample: if sample_category == 'organ' the display name linked to the corresponding description of organ code
-    # otherwise the display name linked to the value of the corresponding description of sample_category code
+    # otherwise sample_category code as the display name for Block, Organ, or Suspension.
```

Comment:
Can we add the missing "Section" to the comment? Just to be complete.

Reply:
@yuanzhou My typo, sorry, but just to confirm, the list is **_only_** Block, Section, or Suspension.  Organ should **_not_** be in the list, as that is previous line documents the return value will be an organ Description from UBKG.

Reply:
Committed without Organ in list, but can change tomorrow, if appropriate.